LibraryTests/SimulatorTestTargets.cs

```diff
@@ -199,19 +199,21 @@ private void LogSimulatorSeed(TestOperation opData, QuantumSimulator sim)
         }
 
         /// <summary>
-        /// Returns a seed to use for the test run if it is stored and null otherwise
+        /// Returns a seed to use for the test run based on the class
         /// </summary>
-        private uint? RetriveStoredSeed(TestOperation opData)
+        private uint? RetriveGeneratedSeed(TestOperation opData)
```

Comment:
typo: RetrieveGeneratedSeed

Reply:
Corrected. Running the test, after pass will commit and push.